templates/targets/riscv64/microchip/polarfire/flake.nix

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  description = "PROJ_NAME - Ghaf based configuration";
+
+  nixConfig = {
+    extra-trusted-substituters = [
+      "https://cache.vedenemo.dev"
+      "https://cache.ssrcdevops.tii.ae"
+    ];
+    extra-trusted-public-keys = [
+      "cache.vedenemo.dev:RGHheQnb6rXGK5v9gexJZ8iWTPX6OcSeS56YeXYzOcg="
+      "cache.ssrcdevops.tii.ae:oOrzj9iCppf+me5/3sN/BxEkp5SaFkHfKTPPZ97xXQk="
+    ];
+  };
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
```

Comment:
This should be github:NixOS/nixpkgs/nixos-23.05 to match the case-sensitivity, although github doesn't seem to care about that

Reply:
Done.